system_strings/sysmes_text.en

```diff
@@ -5673,13 +5673,7 @@ Ciel returns from her visit to the Church's local branch.
 [sha:609ef1fe27daee7a4eecdb58c41f3538a8a0af75]{
 -- ロアの声に逆らうように、^光の中心を目指す。^それは自殺行為に他ならない。
 ロアの声に逆らうように、^光の中心を目指す。^それは自殺行為に他ならない。
-}
-[sha:6a6e8ee21013645a702cbf05c995bbb5cf487998]{
--- それがどうした、と^頭の隅の他人を黙らせる。^ここから戻る選択肢はない。
 それがどうした、と^頭の隅の他人を黙らせる。^ここから戻る選択肢はない。
-}
-[sha:43a21951246fb02775084d4a9c12b0aaf88eebf4]{
--- ロアは語る。^勝ち目が存在しないのなら、^何を“勝ち”とするべきかを。
 ロアは語る。^勝ち目が存在しないのなら、^何を“勝ち”とするべきかを。
 }
```

Comment:
??

Reply:
Hmm must have happened when rebasing on the changed file format